src/parsec.rs

```diff
@@ -44,10 +48,10 @@ const MAX_PARSECS: usize = 10;
 const PARSEC_SIZE_LIMIT: u64 = 1_000_000_000;
 // Limit in integration tests
 #[cfg(all(feature = "mock_base", not(feature = "mock_parsec")))]
-const PARSEC_SIZE_LIMIT: u64 = 200_000;
+const PARSEC_SIZE_LIMIT: u64 = 10_000_000;
 // Limit for integration tests with mock-parsec
 #[cfg(feature = "mock_parsec")]
-const PARSEC_SIZE_LIMIT: u64 = 100;
+const PARSEC_SIZE_LIMIT: u64 = 10_000;
```

Comment:
Why were the 2 limits increased, is that still relevant?
Does that ensure that other tests never prune? is that a good thing. it could be for now anyway.

Reply:
Ah, the increase is to ensure the pruning not happen too much previously (quite long time ago)
However, with the change that we now creates a new parsec instance each time there is a churn.
The change is not relevant anymore. The previous number now only cause pruning with reasonable frequency.
I will change them back.

Reply:
OK, new value seem reasonable, values seem quite high but we can update them if needed.